packages/expo-localization/src/ExpoLocalization.ts

```diff
@@ -2,7 +2,25 @@
 import { Platform } from 'expo-modules-core';
 import * as rtlDetect from 'rtl-detect';
 
-import { Localization } from './Localization.types';
+import { Localization, PreferredCalendar, PreferredLocale } from './Localization.types';
+
+const getNavigatorLocales = () => {
+  return Platform.isDOMAvailable ? navigator.languages || [navigator.language] : [];
+};
+
+type ExtendedLocale = Intl.Locale &
+  // typescript definitions for navigator language don't include some modern Intl properties
+  Partial<
+    {
+      textInfo: { direction: 'ltr' | 'rtl' };
+    } & {
```

Comment:
It seems unnecessary to use `&` here, you can just merge these two objects together 😉